lib/internal/dom-exception.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
```

Comment:
Why are we calling these DOM exceptions?

WebWorkers (assuming that's the behavior we want to mimic) says:

> The term DOM is used to refer to the API set made available to scripts in Web applications, and does not necessarily imply the existence of an actual Document object or of any other Node objects as defined in the DOM Core specifications. [DOM]

I would prefer a different name?

Reply:
The [spec](https://html.spec.whatwg.org/multipage/web-messaging.html#dom-messageport-postmessage) calls it DOMException. The interface is actually specified in [Web IDL](https://heycam.github.io/webidl/#idl-DOMException). So the name isn't up to us to change.

Reply:
To clarify, we could of course use a different name, but then it would make us deviate from browsers in an area that I don't think we need to.

Reply:
+1 to using DOMException. -1 to dash in filename

Reply:
@devsnek Fair enough.